spec/dfe/analytics/send_events_spec.rb

```diff
@@ -45,6 +45,17 @@
       end
     end
 
+    context 'when using fake testing mode' do
+      it 'does not go out to the network' do
+        request = stub_analytics_event_submission
+
+        DfE::Analytics::Testing.fake! do
```

Comment:
This block isn't necessary as it's the default state, set in the body of the `Testing` module (also a la sidekiq) — could rename spec and/or add something to the docs?

Reply:
Yeah it's true that this is default, but I wanted this test to be explicit in case that changes in the future. It's clear that whatever might change around this test, we're at least using the `fake` testing mode in it.